tests/routes/native/v1/offers_test.py

```diff
@@ -64,6 +64,8 @@ def test_get_event_offer(self, app):
             response = TestClient(app.test_client()).get(f"/native/v1/offer/{offer_id}")
 
         assert response.status_code == 200
+        response_content = response.json
+        response_content["stocks"] = sorted(response_content["stocks"], key=lambda stock: stock["id"])
```

Comment:
(micro-détail-promis-demain-j'arrête-de-chipoter)

```suggestion
        response_content["stocks"].sort(key=lambda stock: stock["id"])
```

Reply:
Amusant, j'avais un vague souvenir que c'était deprecated mais je dois confondre.